arch-install.bash

```diff
@@ -1,9 +1,31 @@
 #!/bin/bash
-
 set -euo pipefail
 
 shopt -s extglob globstar nullglob
 
+while getopts "g:i:" opt; do
```

Comment:
i disagree.
I think we should rather remove the -h flag as it is not really needed. You have the usage when you enter wrong paramters and in the readme. It is more important to prevent code duplication IMO

Reply:
Maybe I was being unclear, but I did not suggest duplicating any code.
Removing the `exit` from the function body and exiting with whatever exit code you need whenever it's appropriate does not require duplicated code.

Reply:
I misread the function part segment. Thought you said "This function shouldn't exist". Will change